src/cli/PasswordInput.cpp

```diff
@@ -63,10 +63,15 @@ void PasswordInput::setStdinEcho(bool enable = true)
 QString PasswordInput::getPassword()
 {
     static QTextStream inputTextStream(stdin, QIODevice::ReadOnly);
+    static QTextStream outputTextStream(stdout, QIODevice::WriteOnly);
```

Comment:
@louib maybe nitpicking because the output stream on other cli files don't have the second argument, but I think it's good to merge :)